library/globals.inc.php

```diff
@@ -761,7 +761,7 @@ function gblTimeZones()
         'text_templates_enabled' => array(
             xl('Enable Text Templates in Encounter Forms'),
             'bool',                           // data type
-            '0',                              // default = false
+            '1',                              // default = true
```

Comment:
@sjpadgett ,
Templating stuff is really nice (just tested it out) and definitely think it should be on by default :)
(btw, I finally caught a escaping issue; it had been too long and I was getting worried if I'd ever catch one in your code again :) )
(and the sql upgrade adjustment below if just for the very rare case where somebody may of uninstalled the form_clinical_notes form but kept the form_care_plan form; and vice versa)

![tenor (17)](https://user-images.githubusercontent.com/278968/122029206-8a516f00-cd81-11eb-8d10-13dd1c5f8ec3.gif)